pkg/operator/ceph/cluster/cluster.go

```diff
@@ -397,6 +411,66 @@ func (c *ClusterController) preClusterStartValidation(cluster *cluster, clusterO
 	return nil
 }
 
+func (c *cluster) removeDefaultReplicationRule() error {
+	err := c.context.Executor.ExecuteCommand("ceph", "osd", "crush", "rule", "rm", "replicated_rule")
```

Comment:
Sorry I led you the wrong way, we need to revert this, `git ref` to the rescue! You can still use the context though but not its `Executor`.